src/components/Cashout/Components/Initial.view.tsx

```diff
@@ -77,13 +84,23 @@ export const InitialCashoutView = ({
     const MAX_CASHOUT_LIMIT = 101000 // $101,000 maximum
 
     const isBelowMinLimit = useMemo(() => {
-        return usdValue && parseFloat(usdValue) < MIN_CASHOUT_LIMIT
+        return !usdValue || parseFloat(usdValue) < MIN_CASHOUT_LIMIT
     }, [usdValue])
 
     const isExceedingMaxLimit = useMemo(() => {
         return usdValue && parseFloat(usdValue) > MAX_CASHOUT_LIMIT
     }, [usdValue])
 
+    const isDisabled = useMemo(() => {
+        return (
+            !_tokenValue ||
+            (!selectedBankAccount && !newBankAccount) ||
+            !xchainAllowed ||
+            !!isBelowMinLimit ||
+            !!isExceedingMaxLimit
+        )
+    }, [_tokenValue, selectedBankAccount, newBankAccount, xchainAllowed, isBelowMinLimit, isExceedingMaxLimit])
```

Comment:
intention: fixes bug where the proceeed button wouldn't get enabled even if i added a new account, and seems like better logic